extensions/amp-script/amp-script.md

```diff
@@ -49,7 +49,7 @@ Allows rendering of custom UI components running on third-party JavaScript.
 The `amp-script` component allows you to render widgets and other UI using custom third-party JavaScript, e.g. a React component.
 
 {% call callout('Important', type='caution') %}
-`amp-script` is in active development and under [experimental availability](https://www.ampproject.org/docs/reference/experimental.html). It's subject to breaking API changes and should not yet be used in production.
+`amp-script` is in active development and under [experimental availability](https://amp.dev/documentation/guides-and-tutorials/learn/experimental.html). It's subject to breaking API changes and should not yet be used in production.
```

Comment:
Use the canonical url https://amp.dev/documentation/guides-and-tutorials/learn/experimental

Reply:
fixed